roles/podman/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Allocate subuids for {{ podman_user }} user
+  subuid:
+    user: "{{ podman_user }}"
+  become: true
+  register: subuids
+
+- name: Allocate subgids for {{ podman_user }} user
+  subgid:
+    group: "{{ podman_group }}"
+  become: true
+  register: subgids
+
+- name: Create directory for crun runtime config
+  file:
+    path: /var/lib/{{ podman_user }}/.config/containers
```

Comment:
is this path an actual convention, or was it set up elsewhere?

Reply:
Yep, this was set up by other tasks.  We need to make this path more general.

Reply:
switch to `~{{ podman_user }}/.config/containers`